.github/workflows/ci.yml

```diff
@@ -411,8 +442,13 @@ jobs:
             ~/Library/Caches/Coursier/v1
           key: ${{ runner.os }}-sbt-cache-v2-${{ hashFiles('**/*.sbt') }}-${{ hashFiles('project/build.properties') }}
 
-      - name: Scalafix tests
-        if: matrix.scala == '2.13.8'
-        run: |
-          cd scalafix
-          sbt ci
+      - name: Generate site
+        run: sbt '++${{ matrix.scala }}' site/tlSite
+
+      - name: Publish site
+        if: github.event_name != 'pull_request' && github.ref == 'refs/heads/main'
```

Comment:
I forgot to configure this, but the plan is that the v0.22 site will publish on every push to `series/0.22`